src/Docker.PowerShell/Cmdlets/DockerFactory.cs

```diff
@@ -4,15 +4,26 @@
 using Docker.DotNet;
 using Docker.DotNet.X509;
 
+#if !NET46
+using System.Runtime.InteropServices;
+#endif
+
 public class DockerFactory
 {
     private const string ApiVersion = "1.24";
     private const string KeyFileName = "key.pfx";
     private const string certPass = "p@ssw0rd";
 
+    private static readonly string DefaultHost = (
+#if !NET46
+                    RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? "unix://var/run/docker.sock" :
```

Comment:
Since Windows is the odd duck, maybe change this to check for Windows instead of Linux.